### PR TITLE
load insert in batch

### PR DESCRIPTION
**Pull request by jianwan0214**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #5683 

## What this PR does / why we need it:
load insert in batch

### CODE REVIEW

**Review by daviszhen — APPROVED** [review]

File: pkg/frontend/load_test.go [hunk 1]
```diff
@@ -616,7 +618,7 @@ func Test_rowToColumnAndSaveToStorage(t *testing.T) {
 			}
 			// XXX Vecs[0]?   What are we testing?
 			handler.batchData.Vecs[0] = vector.PreAllocType(Oid[i].ToType(), curBatchSize, curBatchSize, testutil.TestUtilMp)
-			convey.So(rowToColumnAndSaveToStorage(handler, force, row2colChoose), convey.ShouldNotBeNil)
+			convey.So(rowToColumnAndSaveToStorage(handler, proc, force, row2colChoose), convey.ShouldNotBeNil)
```
Comment (m-schen) [inline pkg/frontend/load_test.go:621]: you can add more a check here, to check if the load_test has clean all the memory or not.
`require.Equal(t, proc.mp.now(), proc.mp.old())`
(before that, maybe you need to clean up some bat by yourself, I'm not sure

File: pkg/sql/colexec/auto_incr.go [hunk 2]
```diff
@@ -260,13 +260,40 @@ func updateBatchImpl(ColDefs []*plan.ColDef, bat *batch.Batch, offset, step []ui
...
-	rds, _ := param.rel.NewReader(param.ctx, 1, nil, nil)
-	for {
+func getCurrentIndex(param *AutoIncrParam, colName string, mp *mpool.MPool) (uint64, uint64, error) {
+	ctx := context.TODO()
+
+	var rds []engine.Reader
+
+	ret, err := param.rel.Ranges(ctx, nil)
+	if err != nil {
+		panic(err)
```
Comment (m-schen) [inline pkg/sql/colexec/auto_incr.go:270]: just return the error.

File: pkg/sql/colexec/auto_incr.go [hunk 3]
```diff
@@ -260,13 +260,40 @@ func updateBatchImpl(ColDefs []*plan.ColDef, bat *batch.Batch, offset, step []ui
...
+	}
+	switch {
+	case len(ret) == 0:
+		rds, _ = param.rel.NewReader(ctx, 1, nil, nil)
```
Comment (m-schen) [inline pkg/sql/colexec/auto_incr.go:274]: are you sure we can ignore the error here. please check again

File: pkg/sql/colexec/auto_incr.go (hunk 2 above)
Comment (jianwan0214) [reply pkg/sql/colexec/auto_incr.go]: sure

File: pkg/sql/colexec/auto_incr.go (hunk 3 above)
Comment (jianwan0214) [reply pkg/sql/colexec/auto_incr.go:274]: adjust to return err

File: pkg/frontend/load_test.go (hunk 1 above)
Comment (jianwan0214) [reply pkg/frontend/load_test.go:641]: ok

**Review by m-schen — APPROVED** [review]
lgtm

**Review by nnsgmsone — APPROVED** [review]

**Pull request merged**